W-22202552: Add CloudHub Network Administrator permission note for traffic switching (#881)
mulesoft/docs-runtime-manager: cloudhub/modules/ROOT/pages/app-migration.adoc

[IMPORTANT]
Restart your DLB before initiating the traffic switching.

[NOTE]

> To switch application traffic, you must have the *CloudHub Network Administrator* permission. Without this permission, the traffic switching controls are not available. For more information about assigning permissions, see xref:access-management::permissions-by-product.adoc[Permissions Available in Anypoint Platform].

Reviewer: ```suggestion
To switch application traffic, you must have the CloudHub Network Administrator permission. Without this permission, the traffic switching controls aren't available. For more information about assigning permissions, see xref:access-management::permissions-by-product.adoc[].
```
